app/src/main/java/com/sun/ev_dictionary/data/source/EnglishWordsDataSource.kt

```diff
@@ -1,10 +1,10 @@
 package com.sun.ev_dictionary.data.source
 
-import com.sun.ev_dictionary.data.model.EnglishWord
-import io.reactivex.Single
+import io.reactivex.Observable
 
 interface EnglishWordsDataSource {
     interface Local {
-        fun getWordsFromTextFile(): Single<List<EnglishWord>>
+        fun getWordsFromTextFile(): Observable<Int>
+        fun getNumberOfWords(): Int
```

Comment:
```suggestion
        fun getWordCount(): Int
```
thì có vẻ đúng nghĩa hơn